packages/Uploader/stories/Uploader.stories.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Uploader from "../src/Uploader";
+import Testing from "./Testing";
+
+storiesOf("Uploader", module)
+  .add("Basic example with all files successfully uploaded", () => (
+    <>
```

Comment:
If you use `<Story>` here (`import { Story } from "storybook/assets/styles/common.styles"`) then the story will have a bit of padding.